cluster/gke/util.sh

```diff
@@ -243,7 +243,8 @@ function detect-minion-names {
   detect-node-instance-group
   MINION_NAMES=($(gcloud compute instance-groups managed list-instances \
     "${NODE_INSTANCE_GROUP}" --zone "${ZONE}" --project "${PROJECT}" \
-    | cut -d'/' -f11))
+    --format=yaml | grep instance: | cut -d ' ' -f 2))
```

Comment:
This PR fails unit tests without this change. I can move it to a separate PR, if this change is ok in general.